simple_history/registry_tests/tests.py

```diff
@@ -101,8 +101,8 @@ def test_tracked_abstract_base(self):
                 for f in TrackedWithAbstractBase.history.model._meta.fields
             ],
             [
-                'id', 'history_id', 'history_date',
-                'history_change_reason', 'history_user_id',
+                'id', 'history_id',
+                'history_change_reason', 'history_date', 'history_user_id',
```

Comment:
Why the ordering difference?

Reply:
I'm not sure yet. That was actually a question I had for you?

Reply:
I'll look into it. Not sure why but let me know if you come up with something. I'd want to know why this is happening before merging

Reply:
I think this is happening in `create_history_model` where it stores the history fields in the attrs dict. I think by moving the positional args in the __ini__ method it changed the natural ordering of attrs. I don't think this is a problem. What do you think?